### PR TITLE
added missing storagePath option

### PR DESCRIPTION
**Pull request by GavinCS**
added missing storagePath option for invoiceFile / invoiceDownload in Billable trait.
Update unit tests

### CODE REVIEW

File: src/Laravel/Cashier/Billable.php [hunk 1]
```diff
@@ -105,23 +105,26 @@ public function findInvoiceOrFail($id)
      *
      * @param  string  $id
      * @param  array  $data
+     * @param  null   $storagePath
```
Comment (GrahamCampbell) [inline src/Laravel/Cashier/Billable.php]: wrong

File: src/Laravel/Cashier/Billable.php [hunk 2]
```diff
@@ -105,23 +105,26 @@ public function findInvoiceOrFail($id)
...
     }
 
     /**
      * Create an invoice download Response.
      *
-     * @param  string  $id
-     * @param  array   $data
+     * @param  string $id
+     * @param  array  $data
+     * @param  null   $storagePath
```
Comment (GrahamCampbell) [inline src/Laravel/Cashier/Billable.php]: wrong

File: src/Laravel/Cashier/Billable.php [hunk 3]
```diff
@@ -105,23 +105,26 @@ public function findInvoiceOrFail($id)
...
-        return $this->findInvoiceOrFail($id)->file($data);
+        return $this->findInvoiceOrFail($id)->file($data, $storagePath);
     }
 
     /**
      * Create an invoice download Response.
      *
-     * @param  string  $id
-     * @param  array   $data
+     * @param  string $id
```
Comment (GrahamCampbell) [inline src/Laravel/Cashier/Billable.php]: please revert the alignment changes

File: tests/BillableTraitTest.php [hunk 4]
```diff
@@ -33,11 +33,20 @@ public function testDownloadCallsDownloadOnInvoice()
     {
         $billable = m::mock('BillableTraitTestStub[findInvoice]');
         $billable->shouldReceive('findInvoice')->once()->with('id')->andReturn($invoice = m::mock('StdClass'));
-        $invoice->shouldReceive('download')->once()->with(['foo']);
+        $invoice->shouldReceive('download')->once()->with(['foo'], null);
```
Comment (GrahamCampbell) [inline tests/BillableTraitTest.php]: please don't modify that test

File: src/Laravel/Cashier/Billable.php [hunk 5]
```diff
@@ -105,23 +105,26 @@ public function findInvoiceOrFail($id)
...
+     * @param  array  $data
+     * @param  null   $storagePath
      * @return \Symfony\Component\HttpFoundation\Response
+     * @throws NotFoundHttpException
```
Comment (GrahamCampbell) [inline src/Laravel/Cashier/Billable.php]: incorrect cs - please review the guidelines in our docs

File: tests/BillableTraitTest.php (hunk 4 above)
Comment (GavinCS) [inline tests/BillableTraitTest.php]: this test fails if not updated.  Will this be updated by you?

**Comment by GavinCS** [comment]
updated

**Pull request merged**